ethcore/verification/src/verification.rs

```diff
@@ -187,18 +187,19 @@ fn verify_uncles(block: &PreverifiedBlock, bc: &dyn BlockProvider, engine: &dyn
 			// 5
```

Comment:
Do you think you could clarify this comment a bit for readers unfamiliar with the code?

Reply:
yes, that's a good idea!